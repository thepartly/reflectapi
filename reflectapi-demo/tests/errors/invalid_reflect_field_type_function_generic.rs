#[derive(reflectapi::Input)]
struct MyStruct {
    field: dyn Fn(&str) -> i32,
}

fn main() {}
