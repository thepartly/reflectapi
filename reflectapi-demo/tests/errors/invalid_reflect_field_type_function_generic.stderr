error[E0277]: the trait bound `dyn for<'a> Fn(&'a str) -> i32: Input` is not satisfied
 --> tests/errors/invalid_reflect_field_type_function_generic.rs:3:12
  |
3 |     field: dyn Fn(&str) -> i32,
  |            ^^^^^^^^^^^^^^^^^^^ the trait `Input` is not implemented for `dyn for<'a> Fn(&'a str) -> i32`
